### ISSUE TITLE
Parse failure for paths with a single segment in the d attribute

### ISSUE THREAD

**Issue by Yorwba** [RazrFalcon/resvg#168]
Given the minimal SVG below:
```svg
<svg version="1.1" width="1" height="1" xmlns="http://www.w3.org/2000/svg">
  <path d="M0,0"/>
</svg>
```
`rendersvg` outputs
```
Warning (in usvg::svgtree::parse:63): Failed to parse d value: 'M0,0'.
```
I believe this is due to the [explicit check that a path must consist of at least two segments in `parse.rs`](https://github.com/RazrFalcon/resvg/blob/9183f919c077c0b008e308c2953cc75964c1471a/usvg/src/svgtree/parse.rs#L454), however the [path data grammar](https://www.w3.org/TR/2011/REC-SVG11-20110816/paths.html#PathDataBNF) allows any number of commands. The case where the path is empty is even explicitly called out in the spec.

Given that the path is actually spec-conforming, I think that it shouldn't be treated as a parse error. Do any downstream components depend on the assumption that there are at least two segments?

**Comment by RazrFalcon** [comment]
`resvg` requires a path with at least two commands. Why do you need an empty path?

**Comment by Yorwba** [comment]
I was rendering a bunch of SVGs generated with Adobe Illustrator, which apparently sometimes produces SVG paths like this. When I saw a lot of warning messages, I decided to investigate.

FWIW, I tried simply removing the check and rerendered those SVGs. The only difference was the absence of noisy warnings; the output images had the exact same checksums.

So I don't think `resvg` actually depends on this. That makes sense if you consider that `"M0,0 M0,0"` is essentially the same thing, but doesn't get filtered out by the check.
